docs/_scss/components/_accordions.scss

```diff
@@ -11,21 +11,6 @@ $accordion-border: 3px solid $color-gray-lightest;
     margin-top: 1rem;
```

Comment:
@msecret - Sorry, should have been more explicit.

Yes. The build completed.

I'm not sure what difference there was, but it was enough for `/src/stylesheets/lib` To appear. Not sure why though, as I thought install and update only effected the `/node_modules` directory.